### DISCUSSION TITLE
Welcome to Blender2MSFS2 Discussions!

### DISCUSSION THREAD

**Discussion by tml1024** [tml1024/Blender2MSFS2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mikeaat** [comment]
is it possible to get write access? :)

**Reply by tml1024** [reply]
Sure. A moment while I figure out how to do it. At least you should be able to submit a PR right away.

**Reply by Mikeaat** [reply]
Thanks it worked. I hope i did everything correct. That was my first commit ever :D . For a PR i needed a different branch. Without a Fork, or another Branche direct in this repo that was not possible. :)
